### DISCUSSION TITLE
hw01 step 2

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#5] [Announcements]
Dear All,

Homework 01 step 2 [feedback list](https://github.com/STAT5125-UConn/2022Spring/blob/main/homework/hw01/FeedbackList.md) is released. The deadline for step 2 is postpone to Friday, February 4. Please see the details in ed00f3fb089c2eb183f4ef4213a73670a5900be4

**Comment by Ossifragus** [comment]
The commit is updated to ed00f3f

**Comment by Ossifragus** [comment]
Please give write access to other students to your repository. You can read the instruction [here](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/managing-an-individuals-access-to-an-organization-repository).

**Comment by Boyango** [comment]
Hi Dr. Wang,

I cannot find the “setting” button when I followed the instruction below.  (As shown in the screenshot)
***@***.***

I think the reason might be I do not have permission to change the settings. Could you please check if I have access to the setting please?

Best,
Boyang

**Reply by Ossifragus** [reply]
By my default setting, your role to your repository should be Admin. You are right. Somehow your role became a Reader. I just changed your role to Admin. Please check if it works now.

**Reply by Ossifragus** [reply]
Hi @Boyango
On another matter, I did not see the screenshot your referred. Did you reply the email directly?

**Comment by Boyango** [comment]
Hi Dr. Wang,

Yes I reply the email directly. It’s just a screenshot showing there is no “setting” button. I now have access to the settings and I have sent the invitation. Thank you very much!

Best,
Boyang

**Comment by Ossifragus** [comment]
No problem. Thanks for letting me know.
